app/models/ems_refresh/save_inventory_networks.rb

```diff
@@ -153,12 +153,12 @@ def save_network_routers_inventory(ems, hashes, target = nil)
       store_ids_for_new_records(ems.network_routers, hashes, :ems_ref)
     end
 
-    def save_network_ports_inventory(ems, hashes, target = nil)
+    def save_network_ports_inventory(ems, hashes, target = nil, source = :neutron)
```

Comment:
In the other places we do this, we use the variable `mode` and for inventory we use `:refresh`, and for smartstate we use `:scan`.  If possible, let try to keep the terms consistent. See https://github.com/ManageIQ/manageiq/blob/3f6966555f89b575144f5e6ab69c5682e1165c70/app/models/ems_refresh/save_inventory.rb#L212-L238